api/src/main/java/io/opencensus/trace/Tracer.java

```diff
@@ -192,39 +187,31 @@ public final SpanBuilder spanBuilder(@Nullable Span parent, String name) {
    * <p>This <b>must</b> be used to create a {@code Span} when the parent is in a different process.
    * This is only intended for use by RPC systems or similar.
    *
-   * @param remoteParent The remote parent of the returned Span.
+   * @param remoteParentSpanContext The remote parent of the returned Span.
    * @param name The name of the returned Span.
    * @return a {@code SpanBuilder} to create and start a new {@code Span}.
    * @throws NullPointerException if name is null.
    */
-  public final SpanBuilder spanBuilderWithRemoteParent(
-      @Nullable SpanContext remoteParent, String name) {
-    return SpanBuilder.builderWithRemoteParent(
-        spanFactory, remoteParent, checkNotNull(name, "name"));
-  }
+  public abstract SpanBuilder spanBuilderWithRemoteParent(
```

Comment:
I know, added a comment that users must call with an invalid SpanContext in case of no context received or fail to parse the receiving context, which means we do not accept null remote parent.